cwms/timeseries/timeseries.py

```diff
@@ -106,10 +105,7 @@ def retrieve_ts_df(
             end: datetime, optional, default is None
```

Comment:
Missing documentation on how the timezone is to be used by the date time parameters

Reply:
done

Reply:
I don't see the documentation

Reply:
Oh I see it in the other method, but it should be in both

Reply:
done